ansible/roles/test/tasks/crm/crm_test_acl_counter.yml

```diff
@@ -76,4 +76,4 @@
       assert: {that: "{{new_crm_stats_acl_counter_used|int - crm_stats_acl_counter_used|int == 0}}"}
 
     - name: Verify "crm_stats_acl_counter_available" counter was incremented
-      assert: {that: "{{new_crm_stats_acl_counter_available|int - crm_stats_acl_counter_available|int == 0}}"}
+      assert: {that: "{{new_crm_stats_acl_counter_available|int - crm_stats_acl_counter_available|int >= 0}}"}
```

Comment:
Agree that this is different for certain platforms. But we may want this to be counted properly. How about checking the available counter before add, after add and after delete? Essentially the number of resources that was taken during add, the same must be released after delete leaving this check as it is.

Reply:
@prsunny 
I add the test case to check the available counter before add and after delete.
Please review again.